vault/logical_system.go

```diff
@@ -2955,6 +2956,37 @@ func (b *SystemBackend) handleMetrics(ctx context.Context, req *logical.Request,
 	return b.Core.metricsHelper.ResponseForFormat(format), nil
 }
 
+func (b *SystemBackend) handleInFlightRequestData(_ context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
Both pprof and metrics return their results in a consistent format regardless of whether we're hitting the authenticated or unauthenticated endpoints.  Shouldn't we do the same?

Reply:
No, to the different response format, i.e. the use of the "data" field.